File: discovery-service/src/main/java/com/sudo248/discoveryservice/controller/dto/AddressDto.java
package com.sudo248.discoveryservice.controller.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AddressDto {
    private String addressId;

    private int provinceID = -1;

    private int districtID = -1;

    private int wardCode = -1;

    private String provinceName = "";

    private String districtName = "";

    private String wardName = "";

    private String address = "";

    private double longitude;

    private double latitude;

    private String fullAddress = "";
}

File: client/Soc/app/src/main/java/com/sudo248/soc/data/dto/user/AddressDto.kt
package com.sudo248.soc.data.dto.user

import com.sudo248.soc.domain.entity.user.Location

data class AddressDto(
    val provinceID: Int,
    val districtID: Int,
    val wardCode: String,
    val provinceName: String,
    val districtName: String,
    val wardName: String,
    val address: String,
    val location: Location,
    val fullAddress: String,
)